### ISSUE TITLE
[BUG] Changing Volume Step Size Does Nothing

### ISSUE THREAD

**Issue by Switch123456789** [radj307/volume-control#43]
### Version

5.2.2

### Actual Behaviour

Changing the "volume step" size does nothing, it keeps using the initially loaded value stored in "VolumeControl.json" and also doesn't store the changes in the json-file.

### Expected Behaviour

the changes get applied and stored in the json-file.

### Reproduction Steps

- change the volume step size
- change the volume (won't use set step-size)
- close app & start app again (value is back to the previous one)
### Log

```shell
19:48:33:133  [DEBUG]  VCSettings initializing...
19:48:33:284  [DEBUG]  .ctor is setting the hook source to '723432' (Was null)
19:48:33:284  [DEBUG]  Attached a message hook from 'AddMaximizeBugFixHandler'.
19:48:33:288  [DEBUG]  VCSettings.ExecutablePath = 'D:\DESKTOP\VolumeControl.exe'
19:48:33:295  [DEBUG]  VCSettings.CurrentVersion = '5.2.2'
19:48:33:295  [DEBUG]  VCSettings.ReleaseType = 'NORMAL'
19:48:33:297  [DEBUG]  VCSettings initialization completed.
19:48:33:642  [INFO]   Session selection unlocked.
19:48:33:647  [INFO]   Session selection locked.
19:48:33:650  [DEBUG]  Skipped loading addons because no search directories were located.
19:48:33:651  [DEBUG]  Initialized ActionAddonAttribute
19:48:33:657  [INFO]   Volume Control v5.2.2
19:48:34:058  [DEBUG]  Finished binding event handler method 'HandleShow' to VolumeControl.Audio.AudioAPI events.
19:48:34:074  [DEBUG]  GetIcons called with path '@%SystemRoot%\System32\AudioSrv.Dll' => 'C:\Windows\System32\AudioSrv.Dll'
19:48:34:076  [DEBUG]  GetIcons called with path 'C:\Windows\explorer.exe'
19:48:34:099  [DEBUG]  GetIcons called with path 'C:\Users\Switch\AppData\Local\Discord\app-1.0.9006\Discord.exe'
19:48:34:104  [DEBUG]  GetIcons called with path 'C:\Program Files\NVIDIA Corporation\NvContainer\nvcontainer.exe'
19:48:34:110  [DEBUG]  GetIcons called with path 'C:\ProgramData\chocolatey\lib\ungoogled-chromium\tools\ungoogled-chromium_103.0.5060.134-1.1_windows\chrome.exe'
19:48:34:116  [DEBUG]  GetIcons called with path 'C:\Program Files (x86)\Steam\steam.exe'
19:48:34:122  [DEBUG]  GetIcons called with path 'D:\GAMES\Steam\steamapps\common\Euro Truck Simulator 2\bin\win_x64\eurotrucks2.exe'
19:48:34:127  [DEBUG]  GetIcons called with path 'C:\Users\Switch\AppData\Local\Discord\app-1.0.9006\Discord.exe'
19:48:34:129  [DEBUG]  GetIcons called with path 'C:\Program Files\WindowsApps\SpotifyAB.SpotifyMusic_1.191.824.0_x86__zpdnekdrzrea0\Spotify.exe'
19:48:34:134  [DEBUG]  GetIcons called with path 'C:\Program Files\AntiMicroX\bin\antimicrox.exe'
19:48:37:923  [DEBUG]  GetIcons called with path '%windir%\system32\mmres.dll' => 'C:\Windows\system32\mmres.dll'
19:48:39:498  [INFO]   App exited with code 0
```

**Comment by radj307** [comment]
I figured out the problem; for some reason WPF doesn't care about the databindings for the `ValueProperty` of a `NumericUpDown` unless you explicitly specify `Mode=TwoWay`.

**Comment by radj307** [comment]
I've fixed this locally; I'm going to shift the 5.2.3 tag around so that its included in the release.

**Comment by radj307** [comment]
Fixed in [v5.2.3](https://github.com/radj307/volume-control/releases/tag/5.2.3) and onwards
